### PR TITLE
Api+sync

### PR DESCRIPTION
**Pull request by frankobe**

### CODE REVIEW

File: src/route/api.js [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+apiRouter.post('/last-result-index', (req, res, next) => {
+  DecentralizedOracle.lastResultIndex(req.params)
+    .then((result) => {
+      onRequestSuccess(res, result, next);
+    }, (err) => {
+      onRequestError(res, err, next);
+    });
+});
+
+
```
Comment (dwalintukan) [inline src/route/api.js]: 👾

File: src/route/api.js [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+module.exports = apiRouter;
```
Comment (dwalintukan) [inline src/route/api.js]: EOF line

File: src/route/sync.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+syncRouter.get('/graphql', graphqlRestify(graphQLOptions));
+syncRouter.post('/graphql', graphqlRestify(graphQLOptions));
+
+syncRouter.get('/graphiql', graphiqlRestify({
+  endpointURL: '/graphql',
+  subscriptionsEndpoint: `ws://localhost:${syncPort}/subscriptions`,
+}));
+
+module.exports = syncRouter;
```
Comment (dwalintukan) [inline src/route/sync.js]: EOF line

Comment (frankobe) [reply src/route/sync.js]: done

File: src/route/api.js (hunk 2 above)
Comment (frankobe) [reply src/route/api.js]: done

File: src/route/api.js (hunk 1 above)
Comment (frankobe) [reply src/route/api.js]: done

**Pull request merged**